Makefile

```diff
@@ -34,8 +34,10 @@ build-testaid:
 
 generate-mocks: build-testaid
 	$(GO) get github.com/vektra/mockery/.../
+	rm -f mocks/*.go
```

Comment:
we don't have a clean target yet and I don't see a need to add one just for this. Also, this target is generate mocks and I think it should do so completely. The problem I was seeing was that mocks from non-existent service were still there.